release.sh

```diff
@@ -1,13 +1,9 @@
-#!/usr/bin/env bash
+#!/bin/sh
 
 set -e
 
 mvn scm:check-local-modification
 
-# test
-mvn dependency:tree clean test -Pjdk7,-jdk8
-mvn dependency:tree clean test -Pjdk8,-jdk7
-
 # release
 mvn versions:set
 git add pom.xml
```

Comment:
Undefined :) Basically the one the user is currently using. Shouldn't matter though since we compiled it as target 1.7.